packages/orchestrator-ui-components/src/utils/date.spec.ts

```diff
@@ -29,85 +35,130 @@ describe('date', () => {
+
+            const result = parseDateStringRelativeToToday(kingsDayIsoString);
+
+            const expectedDateTime = `${kingsDayDate}, ${kingsDayTime}`;
+            expect(result).toEqual(expectedDateTime);
+        });
+        it('returns date and time when date is not today', () => {
+            const result = parseDateRelativeToToday(kingsDay);
+
+            const expectedDateTime = `${kingsDayDate}, ${kingsDayTime}`;
+            expect(result).toEqual(expectedDateTime);
+        });
+    });
+
+    describe('parseDateRelativeToToday()', () => {
+        it('returns time only when date is today', () => {
+            const now = new Date();
+            const time = now.toLocaleTimeString(getCurrentBrowserLocale());
+
+            const result = parseDateRelativeToToday(now);
+
             expect(result).toEqual(time);
         });
         it('returns date and time when date is not today', () => {
-            const kingsDay = new Date(kingsDay2023TimeStamp);
-            const date = kingsDay.toLocaleDateString('nl-NL');
-            const time = kingsDay.toLocaleTimeString('nl-NL');
-            const dateTime = `${date} ${time}`;
-            const result = parseDateRelativeToToday(kingsDay.toLocaleString());
-            expect(result).toEqual(dateTime);
+            const result = parseDateRelativeToToday(kingsDay);
+
+            const expectedDateTime = `${kingsDayDate}, ${kingsDayTime}`;
+            expect(result).toEqual(expectedDateTime);
+        });
+        it('returns date only when date is not today and requesting "short notation"', () => {
+            const result = parseDateRelativeToToday(kingsDay, true);
+
+            expect(result).toEqual(kingsDayDate);
         });
     });
+
     describe('calculateTimeDifference()', () => {
         it('returns message with one of the dates missing', () => {
             expect(calculateTimeDifference('', '')).toEqual('');
         });
 
         it('returns appropriate string if the times are the same', () => {
-            const from = new Date(kingsDay2023TimeStamp).toLocaleDateString();
+            const from = new Date(kingsDay2023TimeStamp).toISOString();
             expect(calculateTimeDifference(from, from)).toEqual('00:00:00');
         });
 
         it('returns appropriate string if the timedifference is 1 second', () => {
-            const from = new Date(kingsDay2023TimeStamp).toLocaleString();
-            const to = new Date(kingsDay2023TimeStamp + 1000).toLocaleString();
+            const from = new Date(kingsDay2023TimeStamp).toISOString();
+            const to = new Date(kingsDay2023TimeStamp + 1000).toISOString();
             expect(calculateTimeDifference(from, to)).toEqual('00:00:01');
         });
         it('returns appropriate string if the timedifference is 1 minute', () => {
-            const from = new Date(kingsDay2023TimeStamp).toLocaleString();
+            const from = new Date(kingsDay2023TimeStamp).toISOString();
```

Comment:
I guess parameterized testing (like in pytest) is not supported?

Reply:
Seems to be possible: https://dev.to/tinesoft/be-smart-write-parameterized-tests-with-jest-3eab